worlds/LauncherComponents.py

```diff
@@ -26,13 +26,13 @@ class Component:
     cli: bool
     func: Optional[Callable]
     file_identifier: Optional[Callable[[str], bool]]
-    game_name: Optional[str]
+    game_name: List[str]
     supports_uri: Optional[bool]
 
     def __init__(self, display_name: str, script_name: Optional[str] = None, frozen_name: Optional[str] = None,
                  cli: bool = False, icon: str = 'icon', component_type: Optional[Type] = None,
                  func: Optional[Callable] = None, file_identifier: Optional[Callable[[str], bool]] = None,
-                 game_name: Optional[str] = None, supports_uri: Optional[bool] = False):
+                 game_name: Optional[str | list[str]] = None, supports_uri: Optional[bool] = False):
```

Comment:
```suggestion
                 game_name: Optional[str, List[str]] = None, supports_uri: Optional[bool] = False):
```
though, this only matters if we still support 3.8, which idk if we do at this point

Reply:
Fixed